### ISSUE TITLE
Not rely on Bower

### ISSUE THREAD

**Issue by tcorral** [tcorral/ukko#5]
We should not to use the Bower install directly and we can/should take some parts of bower and create our own "package manager" to install or update the modules.

**Comment by tcorral** [comment]
We can use it but it needs to be refactored.

**Comment by operatino** [comment]
1. `ukko install` should not rely, or interfere with `bower.json` in same directory. Currently if I run `ukko install` in directory that already have Bower, my `bower.json` is changed. Also, if I run the tool without `bower.json` it will prompt an error, since Bower won't run without it.

2. Installing nested component with ukko, it should not process it's Bower dependencies. Right now, if I install `launchpad-ui` (which is a bower package)

```
"bundles/launchpad-ui": "ssh://git@stash.backbase.com:7999/lpm/ui.git"
```

ukko acts like bower, and tries to process nested bower dependencies from `launchpad-ui` component.

**Comment by tcorral** [comment]
Change the don't modify bower.json to not store the installed packages in bower.json and save it in package.json if it is that contains the configuration.

**Comment by tcorral** [comment]
It's needed to check what is bower doing in install but not using it because it reads the bower.json in all the other dependencies and it downloads all the content in the same folder where the original content should exist.  WE DON'T WANT THE DEPENDENCIES, JUST THE CODE THAT WE ARE EXPECTING TO BE DOWNLOADED.

**Comment by tcorral** [comment]
In the example from Robert, the folder ui should be in the root of the folder created with the expected folder to be created but it's inside instead of being the main part of the folder.

**Comment by tcorral** [comment]
Fix in 0.0.23
